src/store/modules/session/index.js

```diff
@@ -5,6 +5,14 @@ export default {
      * @type {String} Stores the theme id.
      */
     theme: '01',
+
+    /**
+     * @type {String} Holds the Google Maps API key for the current instance.
+     */
+    googleMapsApiKey: process.env.NODE_ENV !== 'production'
+      ? 'AIzaSyB0cX-gJYJaVFWnLYLo4WT_BftC7-Yy4Vc'
+      : null,
```

Comment:
todo: we should start to use a .env file (which is not part of the repository itself) in the root of the vue-template to handle secret keys. Vite will support this out of the box. For webpack we can use this package: https://github.com/motdotla/dotenv - See https://stackoverflow.com/a/46232148 for implementation. We can look at this together, since we already define certain variables, so we need to find a different solution for this in the webpack.config.js:

```
new webpack.DefinePlugin({
  "process.env": JSON.stringify(process.env),
}),
```

Reply:
@patric-eberle I have now created the GOOGLE_MAPS_API_KEY variable in .env, works fine. Sure, I would be happy to add more variables together with you.